content/en/docs/tutorials/kubernetes-basics/expose/expose-intro.html

```diff
@@ -109,12 +109,13 @@ <h3>Create a new Service</h3>
 				<p>We have now a running Service called kubernetes-bootcamp. Here we see that the Service received a unique cluster-IP, an internal port and an external-IP (the IP of the Node).</p>
 				<p>To find out what port was opened externally (for the <tt>type: NodePort</tt> Service) we’ll run the <code>describe service</code> subcommand:</p>
 				<p><code><b>kubectl describe services/kubernetes-bootcamp</b></code></p>
-				<p>Create an environment variable called <tt>NODE_PORT</tt> that has the value of the Node port assigned:</p>
-				<p><code><b>export NODE_PORT="$(kubectl get services/kubernetes-bootcamp -o go-template='{{(index .spec.ports 0).nodePort}}')"</b></code><br />
-				<code><b>echo "NODE_PORT=$NODE_PORT"</b></code></p>
+				<p>Create an environment variable called <tt>NODE_IP</tt> that has the value of the Node's IP address and Node port assigned:</p>
```

Comment:
TBH, I don't get the reason behind this change.

Reply:
I have used $NODE_IP for simplicity. I know it's a small change.

But the overall PR makes sense. Or if it doesn't seems good, feel free to close it :smile:.

Reply:
An IP plus a port number cannot be called an IP.